### ISSUE TITLE
File Load Error for etf.hdf5 , etf.hdf5  is not UTF-8 encoded

### ISSUE THREAD

**Issue by cbingos** [jupyterlab/jupyterlab-hdf5#127]
jupyterlab 3.4.3
jupyterlab-hdf==1.2.0

**Comment by loichuder** [comment]
Hello,

Could you provide a file showing the problem so that I can try to find the issue ?

**Comment by cbingos** [comment]
Thank you for your reply：
When I right click menu ：select ——» Open /  Open with ——» Editor:

> File Load Error for etf.hdf5
> etf.hdf5 is not UTF-8 encoded

file: "etf.hdf5"
` import pandas as pd`
`d = {'col1': [1, 2], 'col2': [3, 4]}`
`df = pd.DataFrame(data=d)`
`df.to_hdf('etf.hdf5', key="etf", mode='a', complevel=9, complib="blosc")`
`pd.read_hdf('etf.hdf5','etf')`

**Comment by loichuder** [comment]
> When I right click menu ：select ——» Open / Open with ——» Editor:

Yes, it doesn't work when opening the file this way. Try to double click on it in the file browser.

**Comment by loichuder** [comment]
Note also that [you need to install `hdf5plugin`](https://github.com/jupyterlab/jupyterlab-hdf5#compression-filters) if you use `blosc` to compress your datasets.
